### ISSUE TITLE
Option to use separate name and url (Possibly via raw_url)

### ISSUE THREAD

**Issue by TomRiddle01** [grafana/k6#2950]
### Feature Description

From these threads (https://github.com/grafana/k6/pull/2654, https://github.com/grafana/k6/pull/2726, https://github.com/grafana/k6/issues/2584) and release notes (https://github.com/grafana/k6/releases/tag/v0.40.0, https://github.com/grafana/k6/releases/tag/v0.41.0) I understand that the url and name tags are now essentially the same. My team and I apparently are the minority that uses those tags/values as described by @na-- in https://github.com/grafana/k6/pull/2654#discussion_r983221476.

I did not find an issue regarding the planned `raw_url` (Announced here: https://github.com/grafana/k6/pull/2726#issue-1407404161) so I want to voice interest and maybe start a discussion.

our use case:
The websites we test often have a lot of redirects and dynamic URLs. Because of this we tag all requests with shorter names to group them. In case of unexpected behaviours we used to look up the actual URLs from non-aggregated the InfluxDB/Grafana.
We also like to automatically print a mapping of which names map to which actual URLs to include in a report after the test.

In short: We'd like to continue to use something like the proposed `raw_url` in the mentioned threads.
For our use case this probably means support for non-indexed values in the influxdb output would be needed.

  

### Already existing or connected issues / PRs (optional)

https://github.com/grafana/k6/pull/2654
https://github.com/grafana/k6/pull/2726
https://github.com/grafana/k6/releases/tag/v0.40.0
https://github.com/grafana/k6/releases/tag/v0.41.0

**Comment by TomRiddle01** [comment]
Thanks. We actually do a lot of wrapping already so we probably are equipped to handle changes. Unfortunately we didn't see this ongoing discussion earlier and weren't able to chime in. We aim to share more of our workflow in the future too.

Afaik the "url" tag was an InfluxDB "field" already. From personal experience those should work well with high-cardinality data. We are excited to switch exceptions/errors to high cardinality fields too once user defined high-cardinality metadata is possible. For now we actually use tags for that (Knowing very well, that this can lead to performance issues)

**Comment by sid-maddy** [comment]
> A much easier fix for this problem would be to add a new system `raw_url` "tag" similar to [`vu` and `iter`](https://github.com/grafana/k6/blob/1993ac1b6d47d3504631648d3cf02aebcd5adfbf/metrics/system_tag.go#L37-L38). Similarly to them, `raw_url` will be [disabled by default](https://github.com/grafana/k6/blob/1993ac1b6d47d3504631648d3cf02aebcd5adfbf/metrics/system_tag.go#L44), so there would be no performance implications, and it will not actually be an indexed metric tag, but [high-cardinality non-indexed metric metadata](https://github.com/grafana/k6/blob/1993ac1b6d47d3504631648d3cf02aebcd5adfbf/metrics/system_tag.go#L54).
> 
> This should be simple enough to count for `good first issue` issue tag, and there aren't any big downsides I can see 🤔 You'd have to manually enable it with the [`systemTags` option](https://k6.io/docs/using-k6/k6-options/reference/#system-tags), but it should cover your use case well. @mstoykov and @grafana/k6-core devs, WDYT, 👍 or 👎 (with a comment)?

@na--, thanks for this suggestion! I'm already using this in my fork.  
I can PR it, if this is something that's approved to be included back in k6 core.

**Comment by mstoykov** [comment]
@sid-maddy as long as you also provide tests and preferably at least some rudimentary benchmark results the code from your repo looks okay.
